package.json

```diff
@@ -65,11 +65,11 @@
     "nock": "^11.7.0",
     "prettier": "^1.16.4",
     "proxyquire": "^2.1.3",
-    "semantic-release": "^19.0.0",
+    "semantic-release": "^20.1.0",
     "sinon": "^9.2.1"
   },
   "peerDependencies": {
-    "semantic-release": ">=11.0.0 <20.0.0"
+    "semantic-release": ">=11.0.0 <21.0.0"
```

Comment:
Is there a risk with supporting a too low version of semantic-release here? Probably not right?

Reply:
I don't think so.